src/import/load-costume.js

```diff
@@ -31,7 +33,24 @@ const loadCostumeFromAsset = function (costume, costumeAsset, runtime) {
     if (costumeAsset.assetType === AssetType.ImageVector) {
         // createSVGSkin does the right thing if rotationCenter isn't provided, so it's okay if it's
         // undefined here
-        costume.skinId = renderer.createSVGSkin(costumeAsset.decodeText(), rotationCenter);
+        let svgString = costumeAsset.decodeText();
+        if (costume.version && costume.version === 2) {
+            // SVG Renderer load fixes "quirks" associated with Scratch 2 projects
```

Comment:
Can we move this comment above the if statement and add a note about costume.version here as well (that it should only ever exist for scratch 2 costumes?